applications/eigen_solvers_application/EigenSolversApplication.py

```diff
@@ -0,0 +1,21 @@
+# 
+#   KRATOS _______
+#         / ____(_)___ ____  ____
+#        / __/ / / __ `/ _ \/ __ \
+#       / /___/ / /_/ /  __/ / / /
+#      /_____/_/\__, /\___/_/ /_/ SolversApplication
+#              /____/
+# 
+#   Author: Thomas Oberbichler
+#
+
+from KratosEigenSolversApplication import *
+application = KratosEigenSolversApplication()
+application_name = "KratosEigenSolversApplication"
+application_folder = "eigen_solvers_application"
```

Comment:
I think the naming should be in UpperCamelCase

Reply:
From the Kratos wiki template/script

Reply:
@roigcarlo what is the current rule to follow?

Reply:
Should be in UpperCamelCase if possible indeed. Actually we are just updating the wiki. The new generator does it automatically.